client/src/stories/TextContainer.stories.js

```diff
@@ -0,0 +1,14 @@
+import { TextContainer } from "../components/TextContainer";
+
+export default {
+  title: "Components/TextContainer",
+  component: TextContainer,
+};
+const Template = (args) => <TextContainer {...args} />;
+
+export const TextEmpty = () => <TextContainer />;
```

Comment:
I would make use of the template `args` here too, to allow auto generated Storybook controls

```suggestion
export const TextEmpty = Template.bind({});
```

Reply:
Thanks! Makes sense :)